// @novu/shared/dist/cjs/dto/workflows/preview-step-response.dto.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.GeneratePreviewResponseDto = exports.PreviewPayload = exports.InAppRenderOutput = exports.RedirectTargetEnum = exports.EmailRenderOutput = exports.PushRenderOutput = exports.SmsRenderOutput = exports.ChatRenderOutput = exports.RenderOutput = void 0;
class RenderOutput {
}
exports.RenderOutput = RenderOutput;
class ChatRenderOutput extends RenderOutput {
}
exports.ChatRenderOutput = ChatRenderOutput;
class SmsRenderOutput extends RenderOutput {
}
exports.SmsRenderOutput = SmsRenderOutput;
class PushRenderOutput extends RenderOutput {
}
exports.PushRenderOutput = PushRenderOutput;
class EmailRenderOutput extends RenderOutput {
}
exports.EmailRenderOutput = EmailRenderOutput;
var RedirectTargetEnum;
(function (RedirectTargetEnum) {
    RedirectTargetEnum["SELF"] = "_self";
    RedirectTargetEnum["BLANK"] = "_blank";
    RedirectTargetEnum["PARENT"] = "_parent";
    RedirectTargetEnum["TOP"] = "_top";
    RedirectTargetEnum["UNFENCED_TOP"] = "_unfencedTop";
})(RedirectTargetEnum || (exports.RedirectTargetEnum = RedirectTargetEnum = {}));
class InAppRenderOutput extends RenderOutput {
}
exports.InAppRenderOutput = InAppRenderOutput;
class PreviewPayload {
}
exports.PreviewPayload = PreviewPayload;
class GeneratePreviewResponseDto {
}
exports.GeneratePreviewResponseDto = GeneratePreviewResponseDto;
